tests/stress/stress_test.go

```diff
@@ -111,14 +112,18 @@ func TestSync_SingleBlockProducer(t *testing.T) {
 	}()
 
 	numCmps := 10
+	ctx := context.Background()
+
 	for i := 0; i < numCmps; i++ {
 		time.Sleep(3 * time.Second)
 		t.Log("comparing...", i)
-		hashes, err := compareBlocksByNumberWithRetry(t, nodes, strconv.Itoa(i))
-		if len(hashes) > 1 || len(hashes) == 0 {
-			require.NoError(t, err, i)
-			continue
-		}
+
+		const comparisonTimeout = 5 * time.Second
+		compareCtx, cancel := context.WithTimeout(ctx, comparisonTimeout)
```

Comment:
```suggestion
		const comparisonTimeout = 5 * time.Second
		compareCtx, cancel := context.WithTimeout(ctx, 5 * time.Second)
```
why not inline?

Reply:
Because I got tortured by [`gomnd`](https://github.com/tommy-muehle/go-mnd) for years... So now I constant most of my various 'constant' numbers.